### PR TITLE
SDK-953 Added InApp Controls

### PR DESCRIPTION
**Pull request by darshanclevertap**
Added methods for InApp Controls

Devs can now suspend, discard or resume InApp notifications

### CODE REVIEW

**Review by piyush-kukadiya — APPROVED** [review]
LGTM

File: clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java [hunk 1]
```diff
@@ -1037,6 +1037,35 @@ public Void call() {
                 .getAccountToken() + " accountRegion: " + config.getAccountRegion());
     }
 
+    public void suspendInAppNotifications(){
```
Comment (william-ct) [inline clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java:1040]: If we can write method descriptions for these public methods mentioning the role of `DISCARDED`, `RESUMED` and `SUSPENDED` states, we can clear the confusion especially  `SUSPENDED` and  `DISCARDED` states.

**Review by william-ct — COMMENTED** [review]
LGTM

File: clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java (hunk 1 above)
Comment (darshanclevertap) [reply clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java]: Added Javadoc in latest commit

**Review by william-ct — APPROVED** [review]
LGTM

**Pull request merged**